straxen/plugins/pulse_processing.py

```diff
@@ -185,7 +185,7 @@ def compute(self, raw_records, start, end):
 
         # Do not trust in DAQ + strax.baseline to leave the
         # out-of-bounds samples to zero.
-        # TODO: better to throw an error if something is nonzero
+        # maybe better to throw an error if something is nonzero
```

Comment:
Any intention to change this? I think we can remove the comment entirely.

Reply:
sure